object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/MustCallInvokedChecker.java

```diff
@@ -361,7 +361,11 @@ private void updateDefsWithTempVar(Set<ImmutableSet<LocalVarWithTree>> defs, Nod
                 .toSet();
         defs.remove(setContainingMustCallAliasParamLocal);
         defs.add(newSetContainingMustCallAliasParamLocal);
-      } else if (!(sameResource instanceof LocalVariableNode)) {
+      } else if (!(sameResource instanceof LocalVariableNode
+          || sameResource instanceof FieldAccessNode)) {
+        // we do not track the temp var for the call if the MustCallAlias parameter is a local (that
+        // case is handled above; the local must already be in the defs) or a field (since handling
+        // of @Owning fields is a completely separate check)
```

Comment:
Since this comment specifically mentions owning fields, but the check on line 365 that you've added is checking for any field, it might be worth calling out here that a resource-alias of a non-owning field never needs to be tracked (because it's non-owning).